app/lint.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lint>
+    <issue id="InvalidPackage" severity="ignore"></issue>
```

Comment:
Please use this:

<?xml version="1.0" encoding="UTF-8"?>
<lint>
    <!-- Disable bsh lint -->
    <!-- Error: Invalid package reference in org.apache-extras.beanshell:bsh; not included in Android -->
    <issue id="InvalidPackage">
        <ignore regexp='^.*java.applet..*$'/>
        <ignore regexp='^.*java.awt..*$'/>
        <ignore regexp='^.*javax.script..*$'/>
        <ignore regexp='^.*javax.servlet..*$'/>
        <ignore regexp='^.*javax.swing..*$'/>
    </issue>
</lint>